docs/reference/commandline/ps.md

```diff
@@ -60,5 +61,38 @@ The currently supported filters are:
 
 This shows all the containers that have exited with status of '0'
 
-
-
+## Formatting
+
+The formatting flag (`-F` or `--format`) will pretty-print container output using a Go template.
+
+Valid placeholders for the Go template include:
+* .ID - Container ID
+* .Image - Image ID
+* .Command - Quoted command
+* .CreatedAt - Time when the container was created.
+* .RunningFor - Elapsed time since the container was started.
+* .Ports - Exposed ports.
+* .Status - Container status.
+* .Size - Container disk size.
+* .Labels - All labels asigned to the container.
+* .Label - Value of a specific label for this container. For example `{{.Label "com.docker.swarm.cpu"}}`
+
+The formatting template will either print output exactly as the template
+declares, or with the `table` directive will print the column headers.
+
+The following example uses a template without headers and displays ID and Command:
```

Comment:
"and outputs the `ID` and `Command` of all running containers:"

(s/displays/outputs/ is just a suggestion; I noticed "outputs", "prints" and "displays" were used in this block)